bentoml/_internal/io_descriptors/json.py

```diff
@@ -14,6 +14,11 @@
 from ...exceptions import BadInput
 from ...exceptions import MissingDependencyException
 
+try:
+    import pydantic
+except ImportError:
+    pydantic = None
+
 if TYPE_CHECKING:
     from types import UnionType
```

Comment:
Small thing
Lets use LazyLoader here instead. If user don't want to use pydantic but happens to have pydantic install, bentoml will try to load this.